aws_kms_key.tf

```diff
@@ -4,14 +4,14 @@
 
 
 resource "aws_kms_key" "kmsKeyDisabled" {
-  description         = "KMS key 2"
-  is_enabled          = false
+  description = "KMS key 2"
```

Comment:
**Accurics remediation**
 
**[HIGH]: Customer Master Key (CMK) is disabled. Check if this is intentional.**
 
**Vulnerability**
Identify any disabled AWS KMS Customer Master Keys (CMK) that have been accidentally or intentionally scheduled for deletion in order to prevent losing any data encrypted with these keys.
 
**Remediation**
(From Console) In the navigation pane, choose Customer managed keys. Choose the alias or key ID of a CMK. Under General configuration, choose the Key rotation tab. (AWS Cli) Execute 'aws kms enable-key-rotation --key-id <KMS_KEY_ID>'

source: accurics-